app/assets/javascripts/admin/task/task_create_form_view.js

```diff
@@ -232,8 +222,20 @@ class TaskCreateFormView extends React.PureComponent<Props, State> {
 
               <FormItem label="Experience Domain" hasFeedback>
                 {getFieldDecorator("neededExperience.domain", {
-                  rules: [{ required: true }, { min: 3 }],
-                })(<Input disabled={isEditingMode} />)}
+                  rules: [{ required: true }],
+                })(
+                  <SelectExperienceDomain
+                    disabled={isEditingMode}
+                    mode="default"
+                    placeholder="Select a Experience Domain"
+                    notFoundContent={messages["task.domain_does_not_exist"]}
+                    width={100}
+                    alreadyUsedDomains={[]}
+                    onSelect={null}
+                    onChange={null}
+                    value={null}
```

Comment:
These three lines shouldn't be necessary ideally. Can you omit them? If flow complains, you probably need to add a default value via `static defaultProps = { ...  }` (see here: https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values).